base/src/META-INF/blaze-base.xml

```diff
@@ -387,9 +387,14 @@
     <registryKey defaultValue="false" description="Allow to have optional imports in project view" key="bazel.projectview.optional.imports"/>
     <registryKey
       key="bazel.sync.resolve.virtual.includes"
-      description="Apply heuristics to detect correct location of headers which are accessed from _virtual_includes during build"
+      description="Apply heuristics to detect correct location of headers which are accessed from _virtual_includes during build. (requires re-sync)"
       defaultValue="true"
     />
+    <registryKey
+      key="bazel.sync.clangd.virtual.includes"
```

Comment:
Can we drop clangd here as it's just an implementation detail? I hope some day resharper will also support `-ibazel` aliases